stock_inventory_lockdown/models/stock_location.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# © 2016 Numérigraphe SARL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api, _
+from openerp.exceptions import ValidationError
+
+
+class StockLocation(models.Model):
+    """Refuse changes during exhaustive Inventories"""
+    _inherit = 'stock.location'
+    _order = 'name'
+
+    @api.multi
+    def _check_inventory(self):
+        """Error if an inventory is being conducted here"""
+        location_inventory_open_ids = self.env['stock.inventory'].sudo(
```

Comment:
@mreficent With the change proposed for `_get_locations_open_inventories` the method will be simplified
```python
location_inventory_open_ids = self.env['stock.inventory'].sudo()._get_locations_open_inventories(self.ids)
if location_inventory_open_ids:
    raise ValidationError(...)
```